controllers/constants/ovms-metrics.json

```diff
@@ -0,0 +1,55 @@
+{
+    "metrics": {
+        "supported": true,
```

Comment:
@VedantMahabaleshwarkar you mentioned this needed to be a string value rather than a boolean for technical reasons - is that the case? The dashboard is currently looking for string "true" || "false".

Reply:
Hi Alex, updated the bool to be a string in the second commit! It was an oversight on my part. Apologies.